templates/vhost/_rewrite.erb

```diff
@@ -31,6 +34,9 @@
 <% if @rewrite_rule and !@rewrites -%>
   ## Rewrite rules
   RewriteEngine On
+  <%- if @rewrite_map -%>
+  RewriteMap <%= @rewrite_map %>
+  <%- end -%>
```

Comment:
This block shouldn't be here. It's for backwards compatibility and since the `$rewrite_map` parameter doesn't exist in `apache::vhost` this will never do anything.